### PR TITLE
Add CfP to speakers page

### PR DESCRIPTION
**Pull request by drublic**

### CODE REVIEW

**Comment by drublic** [comment]
screen shot 2017-06-07 at 09 20 27

File: lib/views/pages/speakers.html [hunk 1]
```diff
@@ -31,4 +31,14 @@ <h4 class="speaker_title">Amber Wilson</h4>
             </ul>
         </div>
     </figure>
+
+    <figure id="amberwilson" class="speaker speaker--light">
```
Comment (anselmh) [inline lib/views/pages/speakers.html]: cfp?

File: lib/views/pages/speakers.html [hunk 2]
```diff
@@ -31,4 +31,14 @@ <h4 class="speaker_title">Amber Wilson</h4>
...
+        <img src="/img/speaker/cfp.jpg" alt="Amber Wilson, speaker at NightlyBuild Conference 2017" class="speaker_image" />
```
Comment (anselmh) [inline lib/views/pages/speakers.html]: alt text is wrong…

File: scss/main.scss [hunk 3]
```diff
@@ -587,6 +588,10 @@ input[type='submit'] {
     }
 }
 
+.speaker--light {
+    background: rgba(255,255,255,0.2);
```
Comment (anselmh) [inline scss/main.scss]: can you add spaces?

**Review by anselmh — CHANGES_REQUESTED** [review]

**Comment by drublic** [comment]
@anselmh All done.

**Review by anselmh — APPROVED** [review]

**Pull request merged**